build-tools/Xamarin.Android.Tools.BootstrapTasks/result-packaging.targets

```diff
@@ -0,0 +1,55 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.Zip" />
+  <ItemGroup>
+ <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)Configuration.OperatingSystem.props" Condition="Exists('$(XamarinAndroidSourcePath)Configuration.OperatingSystem.props')" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)Configuration.Override.props" Condition="Exists('$(XamarinAndroidSourcePath)Configuration.Override.props')" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\XABuildConfig.cs" Condition="Exists('$(XamarinAndroidSourcePath)bin\Build$(Configuration)\XABuildConfig.cs')" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\msbuild*.binlog" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)**\config.log" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)**\config.status" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)**\config.h" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)**\CMakeCache.txt" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)**\.ninja_log" />
+    <_BuildStatusFiles Include="$(XamarinAndroidSourcePath)**\android-*.config.cache" />
+  </ItemGroup>
+  <ItemGroup>
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)TestResult-*.xml" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\TestResult-*.xml" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\*.log" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\compatibility\*" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\logcat*" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\msbuild*.binlog*" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\temp\**\*.binlog" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\temp\**\*.log" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\TestOutput-*.txt" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\Timing_*" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)*.csv" />
+    <_TestResultFiles Include="$(TEMP)\llc.exe-*" />
+  </ItemGroup>
+  <PropertyGroup>
+    <ResultPackageOutputPath Condition=" '$(ResultPackageOutputPath)' == '' ">$(XamarinAndroidSourcePath)</ResultPackageOutputPath>
+    <BuildStatusZipName Condition=" '$(BuildStatusZipName)' == '' ">xa-build-status.zip</BuildStatusZipName>
+    <TestResultZipName Condition=" '$(TestResultZipName)' == '' ">xa-test-results.zip</TestResultZipName>
+  </PropertyGroup>
+  <Target Name="ZipBuildStatus"
```

Comment:
My fear with this new approach is that we'll lose the directory structure introduced in commit 781f4b26bdc004e7666c74de69ec545b52a24037, increasing the likelihood that when I click on a `xa-build-status.zip` link, I'll wind up with a bazillion `xa-build-status-N.zip` files, with no "linkage" to what build they're part of, and local `unzip xa-build-status.zip` invocations will result in overwriting/intermixing contents.

Can we preserve the existing naming behavior?

Reply:
I've restored some level of uniqueness to these file names by default. We can now expect result zips to include the version, short revision, OS, and configuration. For example:  ` xa-build-status-9.4.99.48_64264c1-Windows_NT-Release.zip`